notebooks/notebooks/iclr2022/winrate-predictions.ipynb

```diff
@@ -14,16 +14,20 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        np.arange(move_lower_limit, move_upper_limit, move_tick_interval) // 2,
</pre><p>This line confused me for a bit. I think it's because we alternate between adversary and victim moves so only have data points for each of the lines for half of the x-ticks, but want it to plot as a smooth line, so we have the x-ticks be the turn (= move // 2) but still label it by moves? Might be worth adding a comment explaining this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>